includes/admin/settings/class.llms.settings.accounts.php

```diff
@@ -102,6 +102,29 @@ public function get_settings() {
 					'title'   => __( 'Open Registration', 'lifterlms' ),
 					'type'    => 'checkbox',
 				),
+				array(
+					'default'           => 'no',
+					'desc'              => __( 'Discourages account sharing by only allowing the most recent session for each user account.', 'lifterlms' ),
+					'id'                => 'lifterlms_prevent_concurrent_logins',
+					'title'             => __( 'Prevent concurrent logins', 'lifterlms' ),
+					'type'              => 'checkbox',
+					'custom_attributes' => array(
+						'class'         => 'llms-conditional-controller',
+						'data-controls' => '#lifterlms_prevent_concurrent_logins_roles',
+					),
+				),
+				array(
+					'class'             => 'llms-select2',
+					'default'           => array( 'student' ),
+					'desc'              => __( "Users with the selected roles won't be able to be logged in to the same account from different places.", 'lifterlms' ),
+					'id'                => 'lifterlms_prevent_concurrent_logins_roles',
+					'options'           => LLMS_Roles::get_all_role_names(),
+					'title'             => __( 'Prevent concurrent logins for these roles', 'lifterlms' ),
```

Comment:
```suggestion
					'title'             => '',
```
Since this will display below the login option as a second option for the above option I think it looks better in context with the title removed so it's kind of obviously part of the above option:

![Screenshot_2021-12-03_09-12-10](https://user-images.githubusercontent.com/1290739/144635353-afa87880-89c6-4f24-96cf-e9e09b5ed020.png)

What do you think?

Reply:
I think you're right.